### PR TITLE
Backports 0.17 pr13

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: src/qt/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+This directory contains the BitcoinQT graphical user interface (GUI). It uses the cross platform framework [QT](https://www1.qt.io/developers/).
```
Comment (xdustinface) [inline src/qt/README.md:1]:
```suggestion
This directory contains the DashQT graphical user interface (GUI). It uses the cross platform framework [QT](https://www1.qt.io/developers/).
```

File: src/qt/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+The current precise version for QT 5 is specified in [qt.mk](/depends/packages/qt.mk). QT 4 is also supported (see [#8263](https://github.com/bitcoin/bitcoin/issues/8263)).
+
+## Compile and run
+
+See build instructions ([OSX](/doc/build-osx.md), [Windows](/doc/build-windows.md), [Unix](/doc/build-unix.md), etc).
+
+To run:
+
+```sh
+./src/qt/bitcoin-qt
```
Comment (xdustinface) [inline src/qt/README.md:12]:
```suggestion
./src/qt/dash-qt
```

File: src/qt/README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+Resources such as the icon.
+
+### test
+
+Tests.
+
+### bitcoingui.(h/cpp)
+
+Represents the main window of the Bitcoin UI.
```
Comment (xdustinface) [inline src/qt/README.md:35]:
```suggestion
Represents the main window of the Dash UI.
```

File: src/qt/README.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+1. Make sure you installed everything through Homebrew mentioned in the [OSX build instructions](/docs/build-osx.md)
+2. Use `./configure` with the `--enable-debug` flag
+3. In Qt Creator do "New Project" -> Import Project -> Import Existing Project
+4. Enter "bitcoin-qt" as project name, enter src/qt as location
+5. Leave the file selection as it is
+6. Confirm the "summary page"
+7. In the "Projects" tab select "Manage Kits..."
+8. Select the default "Desktop" kit and select "Clang (x86 64bit in /usr/bin)" as compiler
+9. Select LLDB as debugger (you might need to set the path to your installation)
+10. Start debugging with Qt Creator (you might need to the executable to "bitcoin-qt" under "Run", which is where you can also add command line arguments)
```
Comment (xdustinface) [inline src/qt/README.md:95]:
```suggestion
10. Start debugging with Qt Creator (you might need to the executable to "dash-qt" under "Run", which is where you can also add command line arguments)
```

File: src/qt/README.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+A controller. `:NAMEpage.cpp` generally includes `:NAMEmodel.h` and `forms/:NAME.page.ui` with a similar `:NAME`.
+
+### \*dialog.(h/cpp)
+
+Various dialogs, e.g. to open a URL. Inherit from [QDialog](http://doc.qt.io/qt-4.8/qdialog.html).
+
+### paymentserver.(h/cpp)
+
+Used to process BIP21 and BIP70 (see https://github.com/bitcoin/bitcoin/pull/11622) payment URI / requests. Also handles URI based application switching (e.g. when following a bitcoin:... link from a browser).
```
Comment (xdustinface) [inline src/qt/README.md:53]:
```suggestion
Used to process BIP21 and BIP70 (see https://github.com/bitcoin/bitcoin/pull/11622) payment URI / requests. Also handles URI based application switching (e.g. when following a dash:... link from a browser).
```

File: src/qt/README.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+Download and install the community edition of [Qt Creator](https://www.qt.io/download/).
+Uncheck everything except Qt Creator during the installation process.
+
+Instructions for OSX:
+
+1. Make sure you installed everything through Homebrew mentioned in the [OSX build instructions](/docs/build-osx.md)
+2. Use `./configure` with the `--enable-debug` flag
+3. In Qt Creator do "New Project" -> Import Project -> Import Existing Project
+4. Enter "bitcoin-qt" as project name, enter src/qt as location
```
Comment (xdustinface) [inline src/qt/README.md:89]:
```suggestion
4. Enter "dash-qt" as project name, enter src/qt as location
```

**Comment by PastaPastaPasta** [comment]
> And we have the Qt creator instructions twice now but i guess that will be fixed by another backport?
Yeah, looks like that's done in pr14

**Comment by xdustinface** [comment]
> > And we have the Qt creator instructions twice now but i guess that will be fixed by another backport?
> 
> ~Yeah, looks like that's done in pr14~
> It's done in this PR. The qt stuff in build-osx.md is removed

Oh right 😅 rest looks good, utACK!

**Comment by PastaPastaPasta** [comment]
cherry-picked all. LMK if you want me to squash

**Pull request merged**